src/sentry/api/serializers/models/release.py

```diff
@@ -20,7 +20,6 @@ def get_users_for_authors(organization_id, authors, user=None):
     user object exists for that email. If there is no matching
     Sentry user, a {user, email} dict representation of that
     author is returned.
-
```

Comment:
Why these whitespace changes?

Reply:
Sure they were just auto generated by the linter.

Reply:
I didn't know it would do that in a string. That's a little unnerving…